readthedocs/urls.py

```diff
@@ -67,6 +67,7 @@
     url(r'^api/', include(v1_api.urls)),
     url(r'^api/v2/', include('readthedocs.restapi.urls')),
     url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
+    url(r'^api/search/', PageSearchAPIView.as_view()),
```

Comment:
This should likely continue to live in `v2` for now. Perhaps as `api/v2/indocsearch` or something, so it can live beside the old `docsearch` during rollout?

Reply:
Yeah! I added it for testing purpose actually!